docs/general/recording-errors.md

```diff
@@ -5,17 +5,25 @@
 <!-- toc -->
 
 - [What constitutes an error](#what-constitutes-an-error)
+- [Error vs exception](#error-vs-exception)
 - [Recording errors on spans](#recording-errors-on-spans)
 - [Recording errors on metrics](#recording-errors-on-metrics)
+- [Recording errors on logs and events](#recording-errors-on-logs-and-events)
 - [Recording exceptions](#recording-exceptions)
 
 <!-- tocstop -->
 
 This document provides recommendations to semantic convention and instrumentation authors
-on how to record errors on spans and metrics.
+on how to record errors on spans, metrics, and logs.
 
 Individual semantic conventions are encouraged to provide additional guidance.
 
+In the scope of this document, the terms error and exception are defined as follows:
+
+- *Error* refers to a general concept describing any non-success condition. This may include an, non-success status code, or an invalid response.
+
+- *Exception* specifically refers to runtime exceptions and their associated stack traces.
```

Comment:
How does this mesh with languages that don't have "exceptions" and actually only have "errors" like Go or Rust?

Reply:
I refactored it slightly, please take a look, the key change:

added 

> Errors and exceptions are related but not the same: an error can occur without
an exception being thrown, and an exception does not necessarily constitute an
error.
>
> Exceptions and how they are recorded in telemetry are inherently
language-specific. Some languages, such as Rust or Go, do not use exceptions
at all.
>
> This document focuses on recording errors.


Plus there are a couple of points below that should address it

> - Semantic conventions intended for cross-language applicability SHOULD use
>   `error.*` attributes.
> 
> - `exception` attributes SHOULD be documented for conventions that target
>   exceptions directly such as [`exception` events](/docs/exceptions/exceptions-logs.md).